### ISSUE TITLE
(🎁) report PYI042 (or equivalent) in .py files

### ISSUE THREAD

**Issue by KotlinIsland** [charliermarsh/ruff#4386]
Type alias should be Upper case in py files, not just pyi files.

I wonder if there are other PYI rules that should also be applied to .py files 🤔

**Comment by charliermarsh** [comment]
Yeah there's a set of rules that need to be enabled in `.py` files -- there's some existing discussion in #4166.

**Comment by charliermarsh** [comment]
Closed via https://github.com/astral-sh/ruff/pull/6297.
